powerauth-nextstep-model/src/main/java/io/getlime/security/powerauth/lib/nextstep/model/entity/ExternalCredentialDetail.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * PowerAuth Web Flow and related software components
+ * Copyright (C) 2021 Wultra s.r.o.
```

Comment:
If we are insisting on the copyright headers, they should be correct.
```suggestion
 * Copyright (C) 2025 Wultra s.r.o.
```